internal/provider/kubernetes/kubernetes.go

```diff
@@ -87,11 +103,27 @@ func New(ctx context.Context, restCfg *rest.Config, svrCfg *ec.Server, resources
 			mgrOpts.Cache.DefaultNamespaces[watchNS] = cache.Config{}
 		}
 	}
+	if !ptr.Deref(svrCfg.EnvoyGateway.TopologyInjector, egv1a1.EnvoyGatewayTopologyInjector{}).Disable {
+		mgrOpts.WebhookServer = webhook.NewServer(webhook.Options{
```

Comment:
what is the impact of specifying a caCert here than doesnt match the caBundle in the mutating webhook config

Reply:
No requests would be sent. The caBundle here must match the certificate being used in the webhook server and the certificate must also match the target ref in the MutatingWebhookConfiguration (e.g. `envoy-gateway.envoy-gateway-system.svc.local`)